paymentsheet/src/main/java/com/stripe/android/paymentsheet/repositories/StripeCustomerAdapter.kt

```diff
@@ -49,4 +52,26 @@ internal class StripeCustomerAdapter @Inject constructor(
     override suspend fun setupIntentClientSecretForCustomerAttach(): Result<String> {
         TODO()
     }
+
+    internal suspend fun getCustomer(): Result<CustomerEphemeralKey> {
+        return cachedCustomer.takeUnless {
+            it == null || shouldRefreshCustomer()
```

Comment:
You should be able to remove `it == null`, since it's checked in `shouldRefreshCustomer` 

Or maybe I'm misreading.